python/ray/util/client/server/server.py

```diff
@@ -252,14 +254,46 @@ def Terminate(self, req, context=None):
     def GetObject(self, request, context=None):
         return self._get_object(request, "", context)
 
+    def _async_get_object(self, request, client_id: str, req_id: int, context):
+        if request.id not in self.object_refs[client_id]:
+            return ray_client_pb2.GetResponse(valid=False)
+        try:
+            assert request.asynchronous
+            assert context is not None
```

Comment:
do we need these?
What is context here?

Reply:
since these server sides, they will crash the server and the client will just disconnect.

Reply:
No, they are wrapped in a try-Except. They will just be returned to the client side as an error.